client/components/NavBar.tsx

```diff
@@ -1,40 +1,34 @@
 import React from 'react';
-import { Outlet } from 'react-router-dom';
-import Container from 'react-bootstrap/Container';
+
 import Nav from 'react-bootstrap/Nav';
 import Navbar from 'react-bootstrap/Navbar';
 import Button from 'react-bootstrap/Button';
-// import i from 'bootstrap-icons';
+
 
 
 const NavBar = () => {
   const menuItems = ['Map', 'About', 'Feed'];
-  //Check if user is logged in to change text from login to logout
+
   return (
-    <>
-      <Navbar expand='lg' id='mainNavbar'>
-        <Container>
-          <Navbar.Toggle aria-controls='basic-navbar-nav' />
-          <Navbar.Brand href='/'>Hole🕳️Dat</Navbar.Brand>
-          <Button variant='outline-dark' href='/AddPothole' style={{ marginRight: '10%' }}>
-            <i className='bi bi-plus-circle'></i>
-          </Button>
-          <Button href='/auth/google/callback' variant='flat'>
-            Sign In
-          </Button>
-          <Navbar.Collapse id='basic-navbar-nav'>
-            <Nav className='me-auto'>
-              {menuItems.map((item, i) => (
-                <Nav.Link key={i} href={`/${item}`}>
-                  {item}
-                </Nav.Link>
-              ))}
-            </Nav>
-          </Navbar.Collapse>
-        </Container>
-      </Navbar>
-      <Outlet />
-    </>
+    <Navbar expand='lg' id='mainNavbar'>
+      <Navbar.Toggle aria-controls='basic-navbar-nav' />
+      <Navbar.Brand href='/'>Hole🕳️Dat</Navbar.Brand>
+      <Button variant='outline-dark' href='/AddPothole' style={{ marginRight: '10%' }}>
+        <i className='bi bi-plus-circle'></i>
+      </Button>
+      <Button href='/auth/google/callback' variant='flat'>
+        Sign In
+      </Button>
+      <Navbar.Collapse id='basic-navbar-nav'>
+        <Nav className='me-auto'>
+          {menuItems.map((item, i) => (
+            <Nav.Link key={i} href={`/${item}`}>
```

Comment:
it is best practice to use something more specific than just the `index` as the `key` attribute. check out the reasoning in the docs **_[here](https://reactjs.org/docs/reconciliation.html#keys)_**

Reply:
ok I'll fix that